lambda/lambdaUtils.mjs

```diff
@@ -57,8 +57,12 @@ export const invoke = async (functionName, functionArgs, reason) => {
     "x-api-key": process.env.API_KEY,
     "Idempotency-Key": "Idempotency-Key" + Math.floor(Math.random() * 10000),
   };
+  let contractAddress = process.env.LAMBDA_ADDRESS;
+  if (process.env.INVOKE_WITH_PROXY) {
+    contractAddress = process.env.PROXY_ADDRESS;
+  }
```

Comment:
This logic should not be in utils. This is very specific to the application. Send the lambdaAddress in the args

Reply:
removed